src/Castle.Windsor.Extensions.Hosting/Castle.Windsor.Extensions.Hosting.csproj

```diff
@@ -23,7 +23,7 @@
 	</ItemGroup>
 
 	<ItemGroup Condition="'$(TargetFramework)'=='net6.0'">
-    	<PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="6.0.0" />
+    	<PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="8.0.0" />
```

Comment:
should support proper multitarget, we decided that only net8.0 should have 8.x references (see Castle.Windsor.Extensions.DependencyInjection)

Reply:
Fixed